<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCreateModel</class>
 <widget class="QWidget" name="FormCreateModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1678</width>
    <height>918</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="horizontalSpacing">
       <number>9</number>
      </property>
      <property name="verticalSpacing">
       <number>11</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>90</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item row="2" column="6">
          <widget class="QProgressBar" name="creation_progress">
           <property name="value">
            <number>0</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="textVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="5" rowspan="2">
          <widget class="QSpinBox" name="spin_by_quantity">
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QPushButton" name="btn_cancel">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="2">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Model Name:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4" rowspan="2">
          <widget class="QRadioButton" name="rdb_by_quantity">
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Capture images by quantity</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" rowspan="2">
          <widget class="QLineEdit" name="text_model">
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="btn_start">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" rowspan="2">
          <widget class="QPushButton" name="btn_create_model">
           <property name="text">
            <string>Create</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QSpinBox" name="spin_by_time">
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="0" column="3" rowspan="3">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QRadioButton" name="rdb_by_time">
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Capture images by time</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_image_result">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTreeView" name="folders_tree">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>200</number>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
